tests/__snapshots__/test_help_messages.ambr

```diff
@@ -3504,6 +3503,215 @@
+  │                                           value specified for the            │
+  │                                           connection.                        │
+  │ --temporary-connection      -x            Uses connection defined with       │
+  │                                           command line parameters, instead   │
+  │                                           of one defined in config           │
+  │ --mfa-passcode                      TEXT  Token to use for multi-factor      │
+  │                                           authentication (MFA)               │
+  │ --enable-diag                             Run python connector diagnostic    │
+  │                                           test                               │
+  │ --diag-log-path                     TEXT  Diagnostic report path             │
+  │ --diag-allowlist-path               TEXT  Diagnostic report path to optional │
+  │                                           allowlist                          │
+  ╰──────────────────────────────────────────────────────────────────────────────╯
+  ╭─ Global configuration ───────────────────────────────────────────────────────╮
+  │ --format           [TABLE|JSON]  Specifies the output format.                │
+  │                                  [default: TABLE]                            │
+  │ --verbose  -v                    Displays log entries for log levels `info`  │
+  │                                  and higher.                                 │
+  │ --debug                          Displays log entries for log levels `debug` │
+  │                                  and higher; debug logs contains additional  │
+  │                                  information.                                │
+  │ --silent                         Turns off intermediate output to console.   │
+  ╰──────────────────────────────────────────────────────────────────────────────╯
+  
+  
+  '''
+# ---
+# name: test_help_messages[spcs.compute-pool.list]
+  '''
+                                                                                  
+   Usage: default spcs compute-pool list [OPTIONS]                                
+                                                                                  
+   Lists all available compute pools.                                             
+                                                                                  
+  ╭─ Options ────────────────────────────────────────────────────────────────────╮
+  │ --like  -l      TEXT            SQL LIKE pattern for filtering objects by    │
+  │                                 name. For example, `list --like "my%"` lists │
+  │                                 all compute pools that begin with “my”..     │
+  │                                 [default: %%]                                │
+  │ --in            <TEXT TEXT>...  Specifies the scope of this command using    │
```

Comment:
The in scope option doesn't work for compute pool

Reply:
fixed